rp.js

```diff
@@ -0,0 +1,18 @@
+
+const playgame = function play();
```

Comment:
The correct way to declare a function is like this:
`const playGame = function() {
}`
 and always remember to use camelCase when naming variables and functions so `const playgame` should be `const playGame`